wazp/pages/04_dashboard.py

```diff
@@ -1,15 +1,228 @@
+# import base64
+# import pdb
+
+# import pathlib as pl
+# import pdb
+
 import dash
-from dash import html
+
+# import dash_bootstrap_components as dbc
+import pandas as pd
+import plotly.express as px
+
+# import yaml
+from dash import dcc, html  # dash_table
+
+# import wazp.utils as utils
 
 dash.register_page(__name__)
 
+
+##########################
+# Read dataframe for one h5 file---this will be part of the figs' callbacks
+h5_file_path = """/Users/sofia/Documents_local/Zoo_SWC project/WAZP/
+sample_project_2/pose_estimation_results/jwaspE_nectar-open-
+close_controlDLC_resnet50_jwasp_femaleandmaleSep12shuffle1_1000000.h5"""
```

Comment:
Should this path be made relative to the current file or the project repo (as opposed to hard coded on your system)?